scripts/spack/packages/axom/package.py

```diff
@@ -467,10 +487,10 @@ def initconfig_package_entries(self):
         entries.append(cmake_cache_path("CONDUIT_DIR", conduit_dir))
 
         # optional tpls
-        for dep in ("c2c", "mfem", "hdf5", "lua", "raja", "umpire"):
-            if "+%s" % dep in spec:
+        for dep in ("adiak", "caliper", "c2c", "mfem", "hdf5", "lua", "raja", "umpire"):
+            if spec.satisfies("^{0}".format(dep)):
```

Comment:
With this change, if Axom turns off the mfem variant but may still end up with it due to it being somewhere in the spec. I'm not sure that is desired.

Reply:
Fixed